### ISSUE TITLE
Change inputs for Custom PDFs?

### ISSUE THREAD

**Issue by geraintpalmer** [CiwPython/Ciw#107]
Rather than ask for:

```
['Custom`, pdf]
```

Ask for:

```
['Custom`, values, probabilities]
```

This puts it in line with what Numpy asks for. I think it's more intuitive.
An example:

```
'Service_distributions': [['Custom', [[0.1, 9.5], [0.1, 10.2], [0.3, 10.6], [0.2, 10.9], [0.2, 11.7], [0.1, 12.1]]]
```

becomes

```
'Service_distributions': [['Custom', [9.5, 10.2, 10.6, 10.9, 11.7, 12.1], [0.1, 0.1, 0.3, 0.2, 0.2, 0.1]]]
```

Cons: Will change Custom distributions so previous versions are incompatible with newer versions.

**Comment by drvinceknight** [comment]
I like this.

**Comment by geraintpalmer** [comment]
Implemented in #111 

Needs docs. Will change docs once #105 merged in.

**Comment by geraintpalmer** [comment]
Implemented in 69fe5a559903e279672efd48366ecade59aba9aa
